src/m365/aad/commands/o365group/o365group-user-set.ts

```diff
@@ -176,25 +176,23 @@ class AadO365GroupUserSetCommand extends GraphCommand {
     }
   }
 
-  private getOwners(logger: Logger, groupId: string): Promise<User[]> {
+  private async getOwners(groupId: string): Promise<User[]> {
     const endpoint: string = `${this.resource}/v1.0/groups/${groupId}/owners?$select=id,displayName,userPrincipalName,userType`;
 
-    return odata
-      .getAllItems<User>(endpoint)
-      .then(users => {
-        // Currently there is a bug in the Microsoft Graph that returns Owners as
-        // userType 'member'. We therefore update all returned user as owner
-        users.forEach(user => {
-          user.userType = 'Owner';
-        });
+    const users = await odata.getAllItems<User>(endpoint);
+    // Currently there is a bug in the Microsoft Graph that returns Owners as
+    // userType 'member'. We therefore update all returned user as owner
+    users.forEach(user => {
+      user.userType = 'Owner';
+    });
+
+    return users;
 
-        return users;
-      });
   }
 
-  private getMembersAndGuests(logger: Logger, groupId: string): Promise<User[]> {
+  private async getMembersAndGuests(groupId: string): Promise<User[]> {
```

Comment:
You know the drill 😀

Reply:
Haha, I know, but I removed them as they were unused 😂 will readd them.